### DISCUSSION TITLE
How to get Github hosting

### DISCUSSION THREAD

**Discussion by abbasxygum** [community/community#62682] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do we get github to host my website

**Comment by mehdiiwani** [comment, marked as answer]
You can use YourUsername.github.io repository or you can get a pro subscription where you get github pages to host your projects. 

There is also an option education.github.com where you can list as a student and you can get a Pro for free
remember for that you need to add your school email to your account
